cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxHelper.java

```diff
@@ -526,6 +526,13 @@ public static void setStringForKey(String key, String value) {
         editor.commit();
     }
     
+    public static void deleteValueForKey(String key) {
```

Comment:
This function should return bool? So we can tell whether it delete value successfully or not.